README.md

```diff
@@ -35,37 +36,30 @@ var hue = jsHue();
 Then you can discover local bridges:
 
 ```js
-hue.discover(
-    function(bridges) {
-        if(bridges.length === 0) {
-            console.log('No bridges found. :(');
-        }
-        else {
-            bridges.forEach(function(b) {
-                console.log('Bridge found at IP address %s.', b.internalipaddress);
-            });
-        }
-    },
-    function(error) {
-        console.error(error.message);
+hue.discover().then(bridges => {
+    
+    if(bridges.length === 0) {
+        console.log('No bridges found. :(');
     }
-);
+    else {
+        bridges.forEach(function(b) {
+            console.log('Bridge found at IP address %s.', b.internalipaddress);
+        });
+    }
+    
+}
+).catch((err) => {console.log('Error finding bridges', err)});
 ```
 
-jsHue performs requests asynchronously and provides a callback interface. The
-failure callback is called if the XHR request fails, or if there is an error with
-the JSON serialization or deserialization. All API success and error results are
-returned in the success callback data. (This is because some API calls may return
-aggregated API success and error results.)
+jsHue-promise performs requests asynchronously and provides a Promise interface. You can `catch` errors as usual, like in the example below.
 
```

Comment:
@nylki No worries. I would prefer not to do that at this time, because it is possible for the bridge to respond with a mix of successes and failures, like:
```json
[
	{
		"error": {
			"type": 201,
			"address": "/lights/1/state/bri",
			"description": "parameter, bri, is not modifiable. Device is set to off."
		}
	},
	{
		"success": {
			"/lights/1/state/on": false
		}
	}
]
```

and I don't want jsHue to express an opinion on whether that counts as success or failure at the library level, as it's really for the user to say. I'd prefer to keep jsHue's responsibility limited to serialization/deserialization and request/response processing, and pass all results back to the user.